# Generated by Django 4.1.4 on 2022-12-26 20:37

from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('cafe_api', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='Customer',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created_at', models.DateTimeField(default=django.utils.timezone.now, help_text='Specifies when the entity was created.', verbose_name='created at')),
                ('updated_at', models.DateTimeField(default=django.utils.timezone.now, help_text='Specifies when the entity was updated.', verbose_name='updated at')),
                ('phone_number', models.CharField(error_messages={'unique': 'A user with that phone number already exists.'}, help_text="The user's phone number", max_length=12, unique=True, verbose_name='phone number')),
                ('first_name', models.CharField(blank=True, help_text="The user's first name", max_length=150, verbose_name='first name')),
                ('last_name', models.CharField(blank=True, help_text="The user's last name", max_length=150, verbose_name='last name')),
                ('balance', models.FloatField(default=0.0, help_text="The customer's balance", verbose_name='balance')),
            ],
            options={
                'verbose_name': 'customer',
                'verbose_name_plural': 'customers',
            },
        ),
        migrations.AlterField(
            model_name='adminuser',
            name='phone_number',
            field=models.CharField(error_messages={'unique': 'A user with that phone number already exists.'}, help_text="The user's phone number", max_length=12, unique=True, verbose_name='phone number'),
        ),
    ]
